runelite-client/src/main/java/net/runelite/client/plugins/timetracking/farming/FarmingContractInfoBox.java

```diff
@@ -82,9 +82,23 @@ public String getTooltip()
 				contractColor = ColorScheme.PROGRESS_ERROR_COLOR;
 				break;
 			case IN_PROGRESS:
-				contractDescription = "Ready " + TabContentPanel.getFormattedEstimate(manager.getCompletionTime() - Instant.now().getEpochSecond(),
-					config.estimateRelative());
-				contractColor = Color.GRAY;
+				switch (manager.getContractCropState())
+				{
+					case DISEASED:
+						contractDescription = "Diseased";
+						contractColor = Color.YELLOW;
```

Comment:
You should use the exact `CropState` color from enum instead of just defining it as `Color.YELLOW`. Same goes for L93.

Reply:
Ok, I have pushed the requested change.